public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -29,8 +29,12 @@ interface AlertInfo {
   forDuration: string;
   evaluationsToFire: number;
 }
-function ForError({ message }: { message: string }) {
-  return <Badge color="orange" icon="exclamation-triangle" text={'Error'} tooltip={message} />;
+function ForBadge({ message, error }: { message: string; error?: boolean }) {
+  if (error) {
+    return <Badge color="orange" icon="exclamation-triangle" text={'Error'} tooltip={message} />;
```

Comment:
I think we should make error types `red` and use the `exclamation-circle` icon like we do in the alert condition indicator

https://github.com/grafana/grafana/blob/3effd8796463c81f10df264332f53888a5886f92/public/app/features/alerting/unified/components/expressions/AlertConditionIndicator.tsx#L23